### ISSUE TITLE
fisher_shannon on ordinal probability distribution data bug

### ISSUE THREAD

**Issue by MatthewChen37** [arthurpessa/ordpy#10]
Hi, 

First, thank you for your work as I actively use this library in my research. I am plugging the ordinal probability distribution calculated from `ordinal_distribution` into the `fisher_shannon` function with `probs=true`.  However, I get the error:

```

UnboundLocalError: cannot access local variable 'probabilities' where it is not associated with a value

```

in `ord.py` line 2213. This is potentially because `probabilities` is not defined in this case, and unlike `complexity_entropy` I noticed it was missing the lines:

```
probabilities = np.asarray(data)

probabilities = probabilities[probabilities>0]
```

when `probs=true`. This is an implementation bug because `probabilities` should be defined.

**Comment by arthurpessa** [comment]
Thank you for pointing this to us, Mr. Chen.

We are looking into it and will respond to you soon.

**Comment by arthurpessa** [comment]
Good evening, Mr. Chen,

We have fixed the bug you have pointed out in our code by following your suggestion and adding the short line (line 2213): "probabilities = np.asarray(data)". We thank you very much for noticing this mistake and informing us so we could fix it.

Also, upon fixing this bug and doing some housekeeping, we have released a new version of ordpy (version 1.1.4).

Any other mistakes you find or any other questions, please let us know. Best wishes.
